### PR TITLE
Align cli.Dockerfile with akamai/cli release v1.2.0

### PR DESCRIPTION
**Pull request by piotrpio**
This PR should not be merged until akamai/cli is release in version v1.2.0

### CODE REVIEW

File: scripts/push-all.sh [hunk 1]
```diff
@@ -55,7 +55,7 @@ current_build_tags() {
 # HACK : unconditionally push to latest
 #DOCKER_TAG="${DOCKER_TAG} latest"
 
-current_build_tags | grep -Ev 'akamai/(base|cli|.*-chain)' |
+current_build_tags | grep -Ev 'akamai/(base|.*-chain)' |
```
Comment (ynohat) [inline scripts/push-all.sh:58]: @javiergarza could you please confirm that the dockerhub credentials we are using will work with the akamai/cli repository?

**Review by ynohat — APPROVED** [review]
LGTM!
Approved, would simply request @javiergarza take a quick look to validate the akamai/cli dockerhub change.

File: scripts/push-all.sh (hunk 1 above)
Comment (javiergarza) [reply scripts/push-all.sh:58]: I believe so, I can try to keep a look at it tomorrow to ensure the build succeeds

**Review by javiergarza — APPROVED** [review]
Looks good to me

**Review by akamaimike — APPROVED** [review]
LGTM! Approved.

**Pull request merged**